### ISSUE TITLE
Should we use '-' instead of '_' in all command line arguments? 

### ISSUE THREAD

**Issue by astroseger** [singnet/snet-cli#155]
In the current version in some arguments we use '-' and in some '_'.
```
snet organization change-owner
snet client init_channel_metadata
```

@raamb @arturgontijo @ferrouswheel 
Should we use '-' everywhere? I could add it in #148?

**Comment by arturgontijo** [comment]
We should keep a standard for this.
Some popular tools written in python use nothing. (eg `django: startproject and makemigrations`, `scrapy: runspider`)
But for commands like `init_channel_metadata` maybe this won't be good (`initchannelmetadata`).
As I see now we have much more commands with _ than - .
So my vote is to use '_' everywhere.

**Comment by ferrouswheel** [comment]
dashes are much more standard across Unix tools. So :+1: for `-`.

**Comment by astroseger** [comment]
@vforvalerio87 @vsbogd @stellarspot

**Comment by astroseger** [comment]
I will change to "-" in #148. We will have quite a lot of API change. Here script for automatically update shell scripts:
```
FILE=YOU_FILE
sed -i "s/--group_name/--group-name/g" $FILE
sed -i "s/--fixed_price/--fixed-price/g" $FILE
sed -i "s/--service_type/--service-type/g" $FILE
sed -i "s/--metadata_file/--metadata-file/g" $FILE

sed -i "s/snet client account/snet account print/g" $FILE
sed -i "s/snet client balance/snet account balance/g" $FILE
sed -i "s/snet client deposit/snet account deposit/g" $FILE
sed -i "s/snet client withdraw/snet account withdraw/g" $FILE
sed -i "s/snet client transfer/snet account transfer/g" $FILE

sed -i "s/snet service metadata_init/snet service metadata-init/g" $FILE
sed -i "s/snet service metadata_set_fixed_price/snet service metadata-set-fixed-price/g" $FILE
sed -i "s/snet service metadata_add_group/snet service metadata-add-group/g" $FILE
sed -i "s/snet service metadata_add_endpoints/snet service metadata-add-endpoints/g" $FILE
sed -i "s/snet service publish_in_ipfs/snet service publish-in-ipfs/g" $FILE
sed -i "s/snet service update_metadata/snet service update-metadata/g" $FILE
sed -i "s/snet service update_add_tags/snet service update-add-tags/g" $FILE
sed -i "s/snet service update_remove_tags/snet service update-remove-tags/g" $FILE
sed -i "s/snet service print_metadata/snet service print-metadata/g" $FILE
sed -i "s/snet service print_tags/snet service print-tags/g" $FILE
sed -i "s/snet service get_api_metadata/snet service get-api-metadata/g" $FILE
sed -i "s/snet service get_api_registry/snet service get-api-registry/g" $FILE

sed -i "s/snet client open_init_channel_metadata/snet channel open-init-metadata/g" $FILE
sed -i "s/snet client channel_claim_timeout/snet channel claim-timeout/g" $FILE
sed -i "s/snet client channel_extend_add/snet channel extend-add/g" $FILE
sed -i "s/snet client open_init_channel_registry/snet channel open-init/g" $FILE
sed -i "s/snet client print_initialized_channels/snet channel print-initialized/g" $FILE
sed -i "s/snet client print_all_channels/snet channel print-all-filter-sender/g" $FILE

sed -i "s/snet client init_channel_metadata/snet channel init-metadata/g" $FILE
sed -i "s/snet client init_channel_registry/snet channel init/g" $FILE
sed -i "s/snet client block_number/snet channel block-number/g" $FILE
```

**Comment by astroseger** [comment]
was closed in #148
